.travis.yml

```diff
@@ -37,7 +37,7 @@ script:
         cargo rustc --verbose --example ci_test -- -Zno-trans &&
         cargo rustc --verbose --example key_value_store -- -Zno-trans &&
         cargo rustc --verbose -- -Zno-trans &&
-        cargo test  --verbose --features use-mock-crust
+        env RUSTFLAGS="-C codegen-units=8" cargo test --release --verbose --features use-mock-crust --test mock_crust_tests
```

Comment:
Ok, I've made it so that it just does the same thing as before, but with `--release` and `-C codegen-units=8`. It looks like double compiling is unavoidable (due to the need to compile the crate as a test harness **and** a lib).